db/migrate_cards/20180105134605_created_at_for_answers.rb

```diff
@@ -0,0 +1,15 @@
+# -*- encoding : utf-8 -*-
+
+class CreatedAtForAnswers < Card::Migration
+  def up
+    Answer.where(created_at: nil).each do |a|
+      next unless a.card
+      if a.card.type_id == Card::MetricValueID && a.card.researched?
+        a.update_attributes! created_at: a.card.created_at
+      else
+        a.destroy!
```

Comment:
We only want to get rid of the card if the answer is calculated. But we have to set `a.answer_id` to nil. And we probably can't call `delete!`. I expect there is an event somewhere that removes the answer if we do that.

Reply:
OK, good call. I think my new solution handles these. I think 1-3 is obvious. I found the event for #4 and initially altered it, but then I realized that the delete does the lookup via answer_id, so when the answer_id is nil, it won't delete the Answer.